### ISSUE TITLE
Re add Chunk Cache as an option

### ISSUE THREAD

**Issue by I-Tom-I** [RhysB/Project-Poseidon#54]
Please re add the Bukkit Chunk Cache as a server option in poseidon.yml as it helps performance.  I've been on CB1060 for years and noticed it's better with world loading, especially if you have minecart tracks that  go between places that cause chunks to be loaded/unloaded repeatedly.  Uncommenting the lines commented out in ce48b13 and ce48b13 helps with the issue so adding ifs to toggle it via a setting would be great so we don't have to use a fork and need to compile every new version of Poseidon.

**Comment by RhysB** [comment]
I am not keen on adding an option to reenable the default chunk cache due to its significant exploits, allowing players to dupe, bypass permissions, etc. However, I would accept that PR if someone wanted to investigate the chunk cache and resolve the bugs.

I plan to add async chunk loading when I get more time, however, that is likely a while away.

**Comment by I-Tom-I** [comment]
That is why it was suggested as a poseidon.yml option, the default can be off and those who want it can enable it. That way we don't need to fork/merge/recompile Poseidon every time a change is made to keep chunk cache on.

**Comment by CodeKid0** [comment]
I don't think this is really needed since Strultz backported a faster chunk loading fix from modern Minecraft versions. (#61 https://github.com/RhysB/Project-Poseidon/commit/3609ce2cced8cb00af58048dfbd0c3f3f93e3ea7)

**Comment by I-Tom-I** [comment]
It's more of an issue when the hard drive is slow at giving the server data, especially at max view distance.  Traveling between areas on minecarts can put stress on slower hard drives and players will experience slow world loading.

**Comment by CodeKid0** [comment]
> It's more of an issue when the hard drive is slow at giving the server data, especially at max view distance. Traveling between areas on minecarts can put stress on slower hard drives and players will experience slow world loading. Again I'm asking for the _option_, it doesn't need to be a default, because otherwise we'll have to just stick with CB1060.

Well I assume that would use a lot more RAM, and we all know this version of server software is not so good at memory management.
